ui/components/Policies/Utilis/Severity.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Flex from "../../Flex";
+import Icon, { IconType } from "../../Icon";
+import Text from "../../Text";
+
+function Severity({ severity }: { severity: string }) {
+  return (
+    <Flex alignItems="center" gap="4" data-testid={severity}>
+      {(() => {
```

Comment:
Can this be lifted up? A `switch` inside escaped JSX seems unnecessarily complex. Maybe something like this:

```jsx
function Severity() {
        let icon;
        switch (severity.toLocaleLowerCase()) {
          case "low":
            icon = 
              <Icon
                type={IconType.CallReceived}
                color="primary20"
                size="base"
              />
                   ...
        }

    return (
      <Flex>{icon}</Flex>
   )


}
```